mentoris/templates/mentapp/sign_up.html

```diff
@@ -45,47 +45,20 @@ <h2>Welcome, Please Sign Up!</h2>
                 action="{% url 'sign_up' %}"
             >
                 {% csrf_token %}
-                <div class="row mb-3">
-                    <div class="form-group required col-md-6">
-                        <label for="full-name" class="control-label"
-                            >Full Name</label
-                        >
-                        <input
-                            type="text"
-                            class="form-control"
-                            id="full-name"
-                            name="full_name"
-                            placeholder="First Last"
-                            required
-                        />
-                        <div class="invalid-feedback">
-                            Please enter your full name.
-                        </div>
-                    </div>
-                    <div class="form-group required col-md-6">
```

Comment:
Why are we getting rid of the display name? Is that moved somewhere else because I can't see it further down the changes